scripts/tests/service_tests.py

```diff
@@ -917,6 +917,10 @@ def test_apm_server_custom_pipeline(self):
         self.assertIn("foo:/usr/share/apm-server/ingest/pipeline/definition.json", apm_server["volumes"])
         self.assertIn("apm-server.register.ingest.pipeline.overwrite=true", apm_server["command"])
 
+    def test_data_streams(self):
+        apm_server = ApmServer(version="7.16.0", apm_server_enable_data_streams=True).render()["apm-server"]
```

Comment:
Why not allowing from `7.15` on?

Reply:
I've removed any restriction on version number, this is just the one I picked for the test case.